conf/flight_plans/tudelft/delft_bebop_convergence.xml

```diff
@@ -41,35 +41,34 @@
     </sector>
   </sectors>
   <exceptions>
-    <!--<exception cond="!InsideFlight_Area(GetPosX(), GetPosY())" deroute="Holding point"/>-->
-    <exception cond="Or(!InsideFlight_Area(GetPosX(), GetPosY()), GetPosAlt() > GetAltRef() + 20) &&
-      !(IndexOfBlock('Takeoff') >= nav_block) &&
-      !(nav_block >= IndexOfBlock('landed'))" deroute="descend att 0"/>
+    <exception cond="Or(!InsideFlight_Area(GetPosX(), GetPosY()), GetPosAlt() @GT GetAltRef() + 20) &&
```

Comment:
I don't mind to keep the `&&` but it is still better to use `@AND` (and `@OR` instead of `||`) to avoid issues with XML parsers. The one with Ocaml code generator is very permissive, but that might not be the case for the one with the Qt/C++ GCS if you want to use it in the future.